src/mlpack/bindings/R/mlpack/configure

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if [ $(uname) = "SunOS" ]
+then
+sed -i '1 s/$/ -ftrack-macro-expansion=0/' ./src/Makevars
+fi
```

Comment:
That looks promising if a little risky.  No mattern patch?  Just add to the first line?

Reply:
Also, a quick question is, should this be in something like a `Makevars.SunOS` file?  (I think it's fine as-is, that would just make it a bit more consistent with the `Makevars.win` file. :+1:

Reply:
There is code in the R Windows build to call a `Makevars.win`.

There is no code to call a `Makevars.SunOS`.  So while it would look symmetric, all grown-up OSs use `Makevars`   :grinning:

Reply:
Ah, got it, thank you for the clarification. :+1: